### PR TITLE
Ember Times No. 55 - Section about SEO

### PR DESCRIPTION
**Pull request by jessica-jordan**
## What it does

This adds a short section about recent SEO efforts to the next Ember Times.

## Sources

screen shot 2018-07-11 at 15 34 21

### CODE REVIEW

File: source/blog/2018-07-13-the-ember-times-issue-55.md [hunk 1]
```diff
@@ -36,9 +36,22 @@ Check out what's been going in Emberland 🐹 these days:
 
 ---
 
-## [SECTION TITLE](#section-url)
-
-
+## [Find Ember on the Interwebs with the Power of SEO 🔎](#section-url)
+
+These days, weeks and months lots of efforts have been focussing on improving the
```
Comment (Alonski) [inline source/blog/2018-07-13-the-ember-times-issue-55.md]: Would rewrite this as:
These `past` days, weeks and months ... been `focused` on...

File: source/blog/2018-07-13-the-ember-times-issue-55.md [hunk 2]
```diff
@@ -36,9 +36,22 @@ Check out what's been going in Emberland 🐹 these days:
...
+discoverability of official Ember resources including the website, the Guides or the API docs.
+The visibility of these sites to various search engines 🔍 as e.g. DuckDuckGo, Baidu, AOL Search, Dogpile or Google
```
Comment (Alonski) [inline source/blog/2018-07-13-the-ember-times-issue-55.md]: I would replace `as e.g.` to `e.g` or `for example:`

File: source/blog/2018-07-13-the-ember-times-issue-55.md [hunk 3]
```diff
@@ -36,9 +36,22 @@ Check out what's been going in Emberland 🐹 these days:
...
+plays a major role in how easy it is for developers to find the information they are looking for.
+To improve Ember's visibility on the web, Search-Engine Optimization (SEO) of these pages is one of the most effective measures.
+
+Simple, yet effective improvements for Ember sites included tests of functional redirects ([1](https://github.com/emberjs/website/pull/3394)) and fixes for broken links ([2](https://github.com/emberjs/website/pull/3271), [3](https://github.com/emberjs/website/pull/3183)), [4](https://github.com/emberjs/website/pull/3182)).
```
Comment (Alonski) [inline source/blog/2018-07-13-the-ember-times-issue-55.md]: There is an extra `)` at the end here

File: source/blog/2018-07-13-the-ember-times-issue-55.md (hunk 1 above)
Comment (Alonski) [inline source/blog/2018-07-13-the-ember-times-issue-55.md]: I would rewrite this as:
These `past` days, weeks ... been `focusing` on ...

File: source/blog/2018-07-13-the-ember-times-issue-55.md (hunk 2 above)
Comment (Alonski) [inline source/blog/2018-07-13-the-ember-times-issue-55.md]: I would change `as e.g.` to `e.g.` or `for example:`

File: source/blog/2018-07-13-the-ember-times-issue-55.md [hunk 4]
```diff
@@ -36,9 +36,22 @@ Check out what's been going in Emberland 🐹 these days:
...
+Other high-impact efforts included a [new meta data structure for the new Guides](https://github.com/ember-learn/guides-app/pull/135)
+and the continuous evaluation of web crawler data.
+
+If you have previous experience with SEO and you're interested in helping to cast some keyword magic ✨🎩 for Ember sites,
```
Comment (Alonski) [inline source/blog/2018-07-13-the-ember-times-issue-55.md]: For some reason this renders large text and bold for me...

Comment (jessica-jordan) [reply source/blog/2018-07-13-the-ember-times-issue-55.md]: Good catch! It seems a line break to the subsequent `---` and next header title `## Contributors' Corner` was missing here

**Comment by jessica-jordan** [comment]
@Alonski Updated the section according to your suggestions, can you take another look? :eyes:

**Pull request merged**